src/com/shop/Shop.java

```diff
@@ -1,324 +1,335 @@
+	private ResultSet queryResult;
+	private int id;
+	
+	/****************************************/
+	/******** Prepared Statements ***********/
+	/****************************************/
+	private PreparedStatement selectCustomerRow;
+	private PreparedStatement selectSalesRow;
+	private PreparedStatement insertSalesRow;
+	private PreparedStatement selectProductsRow;
+	private PreparedStatement updateBudget;
+	private PreparedStatement updateQuantity;
+	private PreparedStatement selectProductsName;
+	private PreparedStatement selectTotalCost;
+	private PreparedStatement selectAllCustomers;
+	private PreparedStatement insertCustomerRow;
+
+	private void setPreparedStatements(Connection conn) {
+		try {
+			selectCustomerRow = conn.prepareStatement("SELECT * FROM Customers WHERE cid = ?");
+			selectSalesRow = conn.prepareStatement("SELECT * FROM Sales WHERE cid= ? AND pid= ?");
+			insertSalesRow = conn.prepareStatement("INSERT INTO Sales (pid, cid, quantity) VALUES (?, ?, ?)");
+			selectProductsRow = conn.prepareStatement("SELECT * FROM Products WHERE pid= ?");
+			updateBudget = conn.prepareStatement("UPDATE Customers SET budget=budget- ? WHERE cid= ?");
+			updateQuantity = conn.prepareStatement("UPDATE Sales SET quantity=quantity-1 WHERE pid= ? AND cid= ?");
+			selectProductsName = conn.prepareStatement("SELECT * FROM Products WHERE name LIKE ?");
+			selectTotalCost = conn.prepareStatement(
+					"SELECT P.pid AS pid, P.name AS name, P.price*S.quantity AS total FROM Products P, Sales S WHERE S.pid=P.pid AND S.cid= ?");
+			selectAllCustomers = conn.prepareStatement("SELECT * FROM Customers");
+			insertCustomerRow = conn.prepareStatement("INSERT INTO Customers (cid, name, budget) VALUES (?, ?, ?)");
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	private Connection connect() {
+		Connection conn = null;
+		
+		// Connect To Database
+		try {
+			// Load Driver - jdbcDriver
+			Class.forName("oracle.jdbc.OracleDriver");
+			conn = DriverManager.getConnection("jdbc:oracle:***********", "*******", "*******");
+			System.out.println("Connection Successful");
+		} catch (SQLException e) {
+			System.out.println("Connection ERROR");
+			e.printStackTrace();
+		} catch (ClassNotFoundException e1) {
+			e1.printStackTrace();
+		}
+		
+		return conn;
+	}
+
+	private void displayMenu() {
+		System.out.println("\n===== MAIN MENU =====");
+		System.out.println("P - List all products.");
+		System.out.println("O – Order a product.");
+		System.out.println("R – Return a product.");
+		System.out.println("S - Search a product by name.");
+		System.out.println("E - List all orders.");
+		System.out.println("C - Current budget.");
+		System.out.println("X - Exit.");
+		System.out.print("\nEnter Menu Option: ");
+	}
+
+	private boolean isCustomerVerified() {
+		try {
+			selectCustomerRow.setInt(1, id);
+			queryResult = selectCustomerRow.executeQuery();
+			
+			if (queryResult.next()) {
+				return true;
```

Comment:
instead of returning a boolean result variable, I'm just gonna return the boolean value